packages/babel-parser/test/fixtures/jsx/errors/_no-plugin-ts-type-param/options.json

```diff
@@ -0,0 +1,4 @@
+{
+  "throws": "This experimental syntax requires enabling the parser plugin: 'jsx' (1:0)",
```

Comment:
Maybe we could throw `this.expectOnePlugin(["jsx", "flow", "typescript"])` to give an error that is also correct in this case?